react/src/base/inputs/Select.stories.js

```diff
@@ -97,19 +116,34 @@ export const invalidSelectBox = () => (
           ],
         },
       ]}
-      name="name"
-      variant="select"
-      label="Select Box Label"
-      valid={false}
+      name="select"
+      isValid={false}
+    />
+    <SprkIcon
+      iconName="chevron-down"
+      aria-hidden="true"
+      additionalClasses="
+              sprk-c-Icon--filled-current-color
+              sprk-c-Icon--stroke-current-color
+              sprk-b-SelectContainer__icon"
     />
-    <SprkFieldError id="invalid-select">
+<<<<<<< HEAD
+    <SprkFieldError id="select-invalid">
       <SprkIcon
         iconName="exclamation-filled"
         additionalClasses="sprk-b-ErrorIcon"
         aria-hidden="true"
       />
       <div className="sprk-b-ErrorText">There is an error on this field.</div>
     </SprkFieldError>
+=======
+    <SprkErrorContainer
+      id="select-invalid-error"
+      message="Update this story once error container is done"
+    >
+      There is an error on this field.
+    </SprkErrorContainer>
+>>>>>>> add select for
```

Comment:
Got some conflict changes in here to resolve. Removing `SprkErrorContainer`

Reply:
what on earth haha okay, let me see whats up